scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+- group: system_info
+  name: firmware_version
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Indicates the firmware version for the Local Piksi
+  Notes: 'For user generated firmware, this information will appear the same as the
+    GIT command: "git describe --dirty"'
+- group: system_info
+  name: firmware_built
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Date of hardware build
+  Notes:
+- group: system_info
+  name: hw_revision
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: hardware revision for Piksi
+  Notes:
+- group: system_info
+  name: nap_version
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Version of the Navigation Acceleration Process (NAP)
+  Notes: Version of code for the NAP (which runs in the Piksi FPGA)
+- group: system_info
+  name: nap_channels
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Number of tracking channels in the NAP firmware
+  Notes:
+- group: system_info
+  name: nap_fft_index_bits
+  type:
+  units:
+  default value:
+  enumerated possible values:
+  Description: Number of bits to represent the result of fast fourier transform in
+    NAP firmware
+  Notes:
+- group: system_monitor
+  name: heartbeat_period_milliseconds
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Period for sending the SBP_HEARTBEAT and MSG_BASE_POS SBP messages
+  Notes:
+- group: telemetry_radio
+  name: configuration_string
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Configuration string to send radio modem over UART when detected
+  Notes:
+- group: uart_ftdi
+  name: mode
+  type: enum
+  units:
+  default value: SBP
+  enumerated possible values: SBP,NMEA,RTCM
+  Description: Configure mode for uart for the USB port on Piksi
+  Notes:
+- group: uart_ftdi
+  name: sbp_message_mask
+  type: integer
+  units:
+  default value: 65535 (decimal), 0xFFFF (hex)
+  enumerated possible values:
+  Description: Configure the message mask for SBP messages on the uart for the USB
+    port on Piksi
+  Notes: 'The message mask is bitwise anded to the message identifier for a particular
+    message.  If the result is non-zero, the message will be sent over this UART.
+    For example, consider the Piksi firmware sending an SBP
+    message with ID 0x0041. If UART A has mask "64" (0x0040),
+    The SBP subsystem bitwise-ands the message id with the UART A mask giving the result
+    of 0x0040. Since the result is non-zero, the message is valid for UART A and is sent.
+    Practically, the UART with mask 64 (0x0020) transmits
+    only RTK observation data and the USART with mask 65280 (0xFF00) transmits
+    most messages (this is the default USB USART on Piksi).'
```

Comment:
`0xFF00` is intended to transmit most messages of interest to the host system (position, velocity etc., rather than Piksi status)